File: QLNH_Angular/QLNH_Angular/src/app/_models/location.ts
import { GuestTable } from "./guestTable";
import { Restaurant } from "./restaurant";

export interface Location{
    id: number,
    name: string,
    description: string,
    created: Date,
    updated: Date,
    deleted: boolean,
    restaurantId: number,
    restaurant: Restaurant,
    guestTable: GuestTable[]
}

File: QLNH_API/QLNH_Client/Models/Location.cs
using System.ComponentModel.DataAnnotations;

namespace QLNH_Client.Models
{
    public class Location
    {
        [Key]
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public DateTime Created { get; set; } 
        public DateTime Updated { get; set; }
        public bool Deleted { get; set; } = false;
        public int? RestaurantId { get; set; }
        public Restaurant? Restaurant { get; set; }
        public List<GuestTable> GuestTables { get; set; }
    }
}
